lib/ansible/modules/network/cloudengine/ce_config.py

```diff
@@ -240,10 +241,45 @@ def get_running_config(module):
     return NetworkConfig(indent=1, contents=contents)
 
 
+def conversion_src(module):
```

Comment:
There is a bug when commands is run and  source config is a file.
It can not work corrctly.